LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/html/webappapis/dynamic-markup-insertion/opening-the-input-stream/abort-while-navigating.window-expected.txt

```diff
@@ -0,0 +1,16 @@
+CONSOLE MESSAGE: XMLHttpRequest cannot load http://localhost:8800/common/blank.html due to access control checks.
+CONSOLE MESSAGE: Fetch API cannot load http://localhost:8800/common/blank.html due to access control checks.
+
+Harness Error (TIMEOUT), message = null
+
+PASS document.open() aborts documents that are navigating through Location (XMLHttpRequest)
+PASS document.open() aborts documents that are navigating through Location (fetch())
+PASS document.open() aborts documents that are navigating through Location (image loading)
+TIMEOUT document.open() aborts documents that are navigating through iframe loading (XMLHttpRequest) Test timed out
```

Comment:
We do not really like times out. Do you know what the issue is?
If it is due to Safari being specific about iframe failing, can we modify the test?

Reply:
> We do not really like times out. Do you know what the issue is?
> If it is due to Safari being specific about iframe failing, can we modify the test?

I didn't look too much into it because this is only for WK1. As you can see, the test is fully passing for WK2.